privacyidea/models.py

```diff
@@ -2474,3 +2476,204 @@ def __init__(self, username, realm, resolver, authentication,
         self.authentication = authentication
         self.first_auth = first_auth
         self.last_auth = last_auth
+
+### Periodic Tasks
+
+class PeriodicTask(MethodsMixin, db.Model):
+    """
+    This class stores tasks that should be run periodically.
+    """
+    __tablename__ = 'periodictask'
+    id = db.Column(db.Integer, Sequence("periodictask_seq"), primary_key=True)
+    name = db.Column(db.Unicode(), unique=True, nullable=False)
+    active = db.Column(db.Boolean, default=True, nullable=False)
+    interval = db.Column(db.Interval, nullable=False)
+    nodes = db.Column(db.Unicode(256), nullable=False)
+    taskmodule = db.Column(db.Unicode(255), nullable=False)
```

Comment:
length 255? Is there a reason or just coincidence?

Reply:
Ah whoops, this is just by accident. We should probably review the column lengths in general, as e.g. ``name`` doesn't even have a specified length. :)